### ISSUE TITLE
Broken target frames (DragonflightUI)

### ISSUE THREAD

**Issue by Noobsoop** [yuritsuki/WoW-DragonflightUI#1]
The DragonflightUI addon works perfectly fine after loading the game until targeting an elite npc, then all target frames break.

![image](https://github.com/yuritsuki/WoW-DragonflightUI/assets/102838260/bf954a8b-8b61-4756-9643-ac5cf55cee6d)

This is happening on Ascension WoW, allegedly that server is based on patch 3.3.5a and all addons made for that patch should be compatible but there may be some UI changes messing with the addon. If there's something that could be done on my end I'd appreciate to know.

**Comment by yuritsuki** [comment]
Does it happen only on Ascension? Did you try it on blizzlike servers?

**Comment by Noobsoop** [comment]
Yes, I tested on Warmane and that does not happen at all, I don't know if it could happen on any other custom server but it appears to be specific to Ascension.

**Comment by yoaru88** [comment]
Ascension is 3.5.5

**Comment by Noobsoop** [comment]
> Ascension is 3.5.5

There's no such thing as a patch 3.5.5, not even for Classic WotLK. The wiki states Ascension is based on 3.3.5a

**Comment by yoaru88** [comment]
> > Ascension is 3.5.5
> 
> There's no such thing as a patch 3.5.5, not even for Classic WotLK. The wiki states Ascension is based on 3.3.5a

Oh sorry i mean 3.3.5 😂😂
